images/virtualization-artifact/pkg/controller/vm/internal/sync_kvvm.go

```diff
@@ -285,12 +291,12 @@ func (h *SyncKvvmHandler) updateKVVM(ctx context.Context, s state.VirtualMachine
 		return fmt.Errorf("prepare to update KubeVirt VM '%s': %w", current.GetName(), err)
 	}
 
-	if err := h.client.Update(ctx, kvvm); err != nil {
+	if err = h.client.Update(ctx, kvvm); err != nil {
 		return fmt.Errorf("unable to create KubeVirt VM '%s': %w", kvvm.GetName(), err)
 	}
 
-	h.logger.Info("Update KubeVirt VM done", "name", kvvm.Name)
-	h.logger.Debug("Update KubeVirt VM done", "name", kvvm.Name, "kvvm", kvvm)
+	log.Info("Update KubeVirt VM done", "name", kvvm.Name)
+	log.Debug("Update KubeVirt VM done", "name", kvvm.Name, "kvvm", kvvm)
```

Comment:
It looks strange. Are we sure we want to leave it like this?

Reply:
Why not?